src/Cake.Core/Cake.Core.csproj

```diff
@@ -131,11 +131,21 @@
     <Compile Include="IO\IRegistry.cs" />
     <Compile Include="IO\IRegistryKey.cs" />
     <Compile Include="IO\Machine.cs" />
+    <Compile Include="IO\NuGet\AssemblyFilePathFrameworkNameParser.cs" />
```

Comment:
Maybe we should put the parsing classes/interfaces in it's own namespace i.e. `NuGet/Parsing` or something like that.